docs/ABOUT.md

```diff
@@ -1,5 +1,17 @@
 # About
 
+**_COBOL_** is a statically typed, imperative, and procedural programming language with support for object oriented programming. Despite its age, it's still used today in many banking and financial companies due to its default support for fixed-decimal arithmetic and file-processing ability, making it well suited for financial calculations and other business applications. 
+
+First released in 1959, it was developed by the **Committee on Data Systems Languages**(CODASYL) based on the FLOW-MATIC language developed by Remington Rand under the leadership of Real Admiral Grace Hopper. As part of an effort by the United States Department of Defense to create a portable language for processing data, it was adopted by computer manufacturers and then standardized in 1968, with the current standard being ISO 2014. 
+
+The language has two main specialities that seperate it from its peers: fixed-decimal computing, and efficiently processing records(files). These qualities make it perfect for the domains of financial computing and enterprise aaplications, powering everything from payroll systems to airline ticketing. When it comes to processing large amounts of data and large amounts of money, COBOL is on the case.
+
+COBOL was created so that normal businessmen and managers could read and discuss code with programmers, leading to a syntax very close to the english language that prioritizes readability and maintainability. While this mean programs can have a lot of filler words with no semantic meaning and that there are many keywords, properly-written COBOL programs are easily understood, even decades after being written.
+
+**Heads Up:** COBOL is known to be a little intimidating due to its unique syntax and its multitude of keywords. Don't be surprised if something doesn't make sense immediately, and feel free to take breaks. There are also a variety of available resources. Keep in mind there may be some dialect differences between sources:
+
+[IBM COBOL for z/OS][IBM Docs]  
+[TutorialsPoint][Tutorials Point Docs]
```

Comment:
These links don't lead anywhere as the URLs are not defined. If you want to link to external resources, I'd suggest to link to them in a more textual way, as part of a sentence.